.travis.yml

```diff
@@ -1,7 +1,12 @@
 language: python
+dist: xenial
 python:
+  - "3.7"
   - "3.6"
   - "3.5"
```

Comment:
Is there any relevance to specifying more than one python version?

Reply:
Sometimes we get bugs for specific python versions: https://github.com/dnouri/skorch/issues/333

This change ensures we test for 3.7 as well.

Reply:
Sorry, I've overlooked the line installing the travis python version. That makes perfect sense.